batTheme.css

```diff
@@ -2734,4 +2739,4 @@ div::-webkit-scrollbar {
 #GlobalNameHierarchyViewer #GlobalNameHierarchyViewerSelf:hover {
     color: var(--elevated-background-c);
 
-}
\ No newline at end of file
+}
```

Comment:
not sure what happened here, don't remember changing anything at the end of the file